### PR TITLE
feat(contract): add views

### PR DESCRIPTION
**Pull request by Chuck-D-Norris**
This pull request includes significant changes to the `ContractController` and associated views, adding full CRUD (Create, Read, Update, Delete) functionality for contracts. The most important changes include the addition of new methods in the controller, updates to the views for creating and editing contracts, and modifications to the routes to support these new actions.

### Controller Changes:
* [`src/app/Controllers/ContractController.php`](diffhunk://#diff-6bdec0be563d7baff057ed2d8ac4795d1172a1869bd86b85e5895c059a920c52R20-R81): Added methods `create`, `store`, `edit`, `update`, and `destroy` to handle contract creation, storage, editing, updating, and deletion respectively.

### View Changes:
* [`src/app/Views/Contract/Create.php`](diffhunk://#diff-b660c0b23db00cc87b8468ad1ae1517f77e0ac7b7d873ff41ff35d6676826973R1-R71): Added a new view for creating contracts, including a form with fields for contract details such as name, start date, end date, and invoice amounts.
* [`src/app/Views/Contract/Edit.php`](diffhunk://#diff-f34a9bd609954d6cbb2d9541de90c51444cc6ad1139f1894bc7a25cdfcf3f365R1-R70): Added a new view for editing contracts, pre-filled with the existing contract data and including a form for updating contract details.
* [`src/app/Views/Contracts.php`](diffhunk://#diff-ecc65e34880927879bab76650044477bd764cbe952fca4fb72cc39aeb9867647L15-R17): Updated the contracts list view to include a "Create Contract" button and added action buttons for editing and deleting each contract. [[1]](diffhunk://#diff-ecc65e34880927879bab76650044477bd764cbe952fca4fb72cc39aeb9867647L15-R17) [[2]](diffhunk://#diff-ecc65e34880927879bab76650044477bd764cbe952fca4fb72cc39aeb9867647R33-R84)

### Route Changes:
* [`src/routes/web.php`](diffhunk://#diff-97834c4cd54300b04a6c52ee47d862985b78d90cb2fe3b7662b1208f2fc21afdR183-R197): Added new routes to handle the creation, editing, updating, and deletion of contracts, ensuring these actions are protected by authentication middleware. [[1]](diffhunk://#diff-97834c4cd54300b04a6c52ee47d862985b78d90cb2fe3b7662b1208f2fc21afdR183-R197) [[2]](diffhunk://#diff-97834c4cd54300b04a6c52ee47d862985b78d90cb2fe3b7662b1208f2fc21afdR271-R281)

### CODE REVIEW

**Review by 0x1026 — APPROVED** [review]

**Review by elbacalau — APPROVED** [review]

File: src/app/Views/Contract/Create.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            <label for="invoice_proposed" class="block text-sm font-medium text-gray-700 mb-1">Invoice Proposed</label>
+            <input type="number" step="0.01" id="invoice_proposed" name="invoice_proposed"
+                class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring focus:ring-blue-500 focus:border-blue-500"
+                required>
+        </div>
+
+
+        <div>
+            <label for="invoice_agreed" class="block text-sm font-medium text-gray-700 mb-1">Invoice agreed</label>
+            <input type="text" id="invoice_agreed" name="invoice_agreed"
```
Comment (0x1026) [inline src/app/Views/Contract/Create.php:56]:
```suggestion
            <input type="number" id="invoice_agreed" name="invoice_agreed"
```

File: src/app/Views/Contract/Create.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring focus:ring-blue-500 focus:border-blue-500"
+                required>
+        </div>
+
+
+
+        <div>
+            <label for="invoice_paid" class="block text-sm font-medium text-gray-700 mb-1">Invoice paid</label>
+            <input type="text" id="invoice_paid" name="invoice_paid"
```
Comment (0x1026) [inline src/app/Views/Contract/Create.php:65]:
```suggestion
            <input type="number" id="invoice_paid" name="invoice_paid"
```

File: src/app/Views/Contract/Create.php [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                required>
+        </div>
+
+
+
+        <!-- Start Date -->
+
+        <div>
+            <label for="start_date" class="block text-sm font-medium text-gray-700 mb-1">Start Date</label>
+            <input type="text" id="start_date" name="start_date"
```
Comment (0x1026) [inline src/app/Views/Contract/Create.php:30]:
```suggestion
            <input type="date" id="start_date" name="start_date"
```

File: src/app/Views/Contract/Create.php [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring focus:ring-blue-500 focus:border-blue-500"
+                required>
+        </div>
+
+        <!-- End Date -->
+
+
+        <div>
+            <label for="end_date" class="block text-sm font-medium text-gray-700 mb-1">End Date</label>
+            <input type="text" id="end_date" name="end_date"
```
Comment (0x1026) [inline src/app/Views/Contract/Create.php:40]:
```suggestion
            <input type="date" id="end_date" name="end_date"
```

File: src/app/Views/Contract/Create.php [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+
+        <div>
+            <label for="created_ad" class="block text-sm font-medium text-gray-700 mb-1">Created ad</label>
+            <input type="text" id="created_ad" name="created_ad"
+                class="w-full px-4 py-2 border border-gray-300 rounded-lg focus:outline-none focus:ring focus:ring-blue-500 focus:border-blue-500"
+                required>
+        </div>
+
+
```
Comment (0x1026) [inline src/app/Views/Contract/Create.php:79]:
```suggestion
```

**Review by 0x1026 — CHANGES_REQUESTED** [review]
As we talked at class, you missed to commit these changes!

**Review by 0x1026 — APPROVED** [review]

**Pull request merged**